packages/adapters/wagmi/src/connectors/AuthConnector.ts

```diff
@@ -67,7 +68,14 @@ export function authConnector(parameters: AuthParameters) {
             AlertController.open(ErrorUtil.ALERT_ERRORS.UNVERIFIED_DOMAIN, 'error')
           }
         },
-        abortController: ErrorUtil.EmbeddedWalletAbortController
+        abortController: ErrorUtil.EmbeddedWalletAbortController,
+        getActiveCaipNetwork: (namespace?: ChainNamespace) => {
+          if (namespace) {
+            return ChainController.getActiveCaipNetwork(namespace)
+          }
+
+          return ChainController.state.activeCaipNetwork
```

Comment:
can' we make `getActiveCaipNetwork` have this behavior directly so we don't have to repeat the code?

Reply:
I believe that's the current behavior already